server/interaction/calculated_npc.go

```diff
@@ -0,0 +1,22 @@
+package interaction
+
+import (
+	"github.com/joshprzybyszewski/cribbage/logic/strategy"
+	"github.com/joshprzybyszewski/cribbage/model"
+)
+
+var _ npcLogic = (*calcNPCLogic)(nil)
+
+type calcNPCLogic struct{}
```

Comment:
🌵 the word logic is scary to me. I'd prefer `calculatedNPC` because "an NPC" is a "piece of data" whereas a "logic" is a behavior. I want to try to keep our code data-oriented as much as possible and avoid "brain classes"